.github/workflows/pypi_publish.yml

```
@@ -12,11 +12,11 @@ on:
 
 jobs:
   wait-for-approval:
-    uses: datachecks/github-action-workflows/.github/workflows/manual_approver.yaml@main
+    uses: .github/workflows/manual_approver.yaml
```

actionlint (1.7.7)

reusable workflow call ".github/workflows/manual_approver.yaml" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)